html/pages/deluser.inc.php

```diff
@@ -14,7 +14,7 @@
             $delete_username = dbFetchCell('SELECT username FROM users WHERE user_id = ?', array($vars['id']));
 
             if ($vars['confirm'] == 'yes') {
-                if (deluser($delete_username)) {
+                if (deluser($vars['id']) >= 0) {
```

Comment:
I was thinking of the function.   What do you want to know upon return of the function.  Was it successful or not.  (rather than the id that you already sent or not if if it failed).